engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -13,9 +15,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends WorkflowStore {
   override def initialize(implicit ec: ExecutionContext): Future[Unit] = {
-    sqlDatabase.updateWorkflowState(
-      WorkflowStoreState.Running.toString,
-      WorkflowStoreState.Restartable.toString)
+          if ( ConfigFactory.load().as[Option[Boolean]]("system.workflow-restart").getOrElse(true)) {
+            sqlDatabase.updateWorkflowState(
+                WorkflowStoreState.Running.toString,
+                WorkflowStoreState.Restartable.toString)
+          } else {
+            Future.successful(())
```

Comment:
It's a fair question.  Yes -- I think that's what should happen.  Predominantly because if you've been running one way for a long time and then fiddle some "advanced" setting (ie only documented in the reference.conf), you should know what you're doing ;)  We do have a need to disable this for the use case I mentioned above, so I want to be able to support that, which is impossible now since it's hard coded, but not solve for use cases that we don't have right now because we're unlikely to get it right in absence of a such a use case.  phew!